### DISCUSSION TITLE
Could something like this work on the Qualcomm Pixels?

### DISCUSSION THREAD

**Discussion by Cool-Katt** [kyujin-cho/pixel-volte-patch#67] [Ideas]
Title pretty much says it all, could something like this be done to get VoLTE working on the qualcomm pixels, 5 and prior? I'm not familiar with the mechanism of the lock so I don't know how applicable it is. I know that there are some solutions that require rooting, but I'm interested in something like this, which doesn't.

**Comment by silbaums** [comment]
hey @Cool-Katt for me this worked on even no pixel phone oneplus 10 android 13, 
all good give it a try

**Reply by Cool-Katt** [reply]
Yeah, I've tried the app too and it does unhide the menus in settings but it doesn't seem to actually enable any VoLTE/VoWIFI service. 
It is possible it's also a carrier issue, since my caried doesn't officially support any pixel, but i do know for a fact that on tensor ones this method works, for my carrier too.

**Reply by silbaums** [reply]
@Cool-Katt sometimes also MNO(Operator) are whitelisting devices from their side, try your sim in Operators suggested device aka some Samsung or iPhone if then sim have VoLTE and your device don't have volte even with slider turned on then more things need to be changed

**Reply by fillwithjoy1** [reply]
Just double checking, did you say that your carrier has enabled support for Google devices in general? What carrier are you using btw

**Reply by silbaums** [reply]
LMT, Tele2 and Bite 
ASLso Telia and Elisa 

Plus 5G started to work without root

**Reply by fillwithjoy1** [reply]
Okay, I'll supply my user agent assuming the format for your carrier is `#MANUFACTURER#_#MODEL#_Android#AV#_#BUILD#`

User Agent: `Google_Pixel 6 Pro_Android 14_UQ1A.240205.002`

If that works, thumbs up or leave a comment on the #274 pull request so I can get this added within the app itself

**Comment by fillwithjoy1** [comment]
Yes this should (in theory) work across all Androids as it modifies carrier files that are visible across all Androids. It's just that OEMs customize certain things, like the themes of the phone and maybe they might like to change how the phone connects to your carrier which then is an issue that we can't fix as we don't know the inner workings of that customized OS. 
I don't know why but I believe the reason kyochin specified `Tensor-enabled devices` is due to those devices currently receiving the latest versions of android.

**Comment by silbaums** [comment]
Yes now in Latvia all operators have volte enabled by default without a patch
starting from the Android 14 December release
Not full codec support but still volte works out of box from Google.

Same in Estonia 

I need to check how it's in Lithuania, with default device config

**Comment by peerless2012** [comment]
Pixel 5 with LOS 22 (Android 15) , works in China (China Unicom).
